syntax = "proto3";

package where_child_bus;

service HealthcheckService {
  rpc Ping(PingRequest) returns (PingResponse);
}

message PingRequest {
  string name = 1;
}

message PingResponse {
  string message = 1;
}
